src/components/FlowChart/FlowChart.tsx

```diff
@@ -32,15 +36,15 @@ export default function FlowChart(props: Props) {
 
   const reactFlowWrapper = useRef<HTMLElement>(null)
 
-  const onConnect = (params: Edge | Connection) => {
+  const onConnect = (params: Link | NodeConnection) => {
```

Comment:
The idea behind this was to keep typing consistent.

We have something like this

export type FlowNode = Node<NodeData>
export type Link = Edge

So we have a custom Node type FlowNode. If we would use Edge, you would have to import FlowNode from us but the Edge from the library
In order to import from one source of truth, I thought It might make sense to define only custom types so that you don't have to import arbitrarily from us and from the library

Reply:
I get your point, I think in that case, I feel that it's better to name it `FlowEdge`

Then we have `FlowNode` and `FlowEdge` which signifies that they're connected (:stuck_out_tongue_winking_eye: )